api.js

```diff
@@ -42,6 +44,17 @@ module.exports = async function attachAPI(app, {wss, db}) {
     }
   }
 
+  const md5 = string => {
+    if (!string) {
+      throw 'md5() was not passed ' + string
+    }
+
+    return crypto.createHash('md5').update(string).digest('hex')
+  }
+
+  const emailToAvatarURL = memoize(email =>
+    `https://seccdn.libravatar.org/avatar/${email ? md5(email) : ''}?d=retro`)
+
```

Comment:
This will cause mixed content warnings on browsers using a client hosted on `http` (not secure), I think? Not a big deal, but it would be nice to set this based on whether or not the server's HTTPS setting is true or not.

Reply:
No, it's the other way around. But I agree.

Reply:
Ah, okay, cool!